compiler-rt/lib/sanitizer_common/sanitizer_platform.h

```diff
@@ -318,6 +318,9 @@
 #  endif
 #endif
 
+// The beginning of mmap range of addresses.
```

Comment:
_nit_: the pre-existing verbiage for `SANITIZER_MMAP_RANGE_SIZE` reads: 

> "// The range of addresses which can be returned my mmap."

So I think it would be nice to keep a similar phrasing, so I recommend:

```suggestion
// The first address that can be returned by mmap
```